Common/src/main/resources/assets/crafttweaker/lang/it_it.json

```diff
@@ -0,0 +1,103 @@
+  "crafttweaker.command.description.info.block" : "Attiva o disattiva il lettore di blocchi. In modalità di analisi dei blocchi, facendo clic con il tasto destro su un blocco potrai visualizzare il nome, i metadati e i dati dell'entità-blocco di quel blocco.",
+  "crafttweaker.command.description.info.entity" : "Attiva o disattiva il lettore di entità. In modalità di analisi delle entità, facendo clic con il tasto destro su un'entità potrai visualizzare il nome e i dati di quell'entità.",
+  "crafttweaker.command.description.inventory" : "Restituisce i nomi degli oggetti nel tuo inventario",
+  "crafttweaker.command.description.inventory.tags" : "Restituisce i tag degli oggetti nel tuo inventario",
+  "crafttweaker.command.description.issues" : "Apre un link al registro bug",
+  "crafttweaker.command.description.log" : "Apre il file del log",
+  "crafttweaker.command.description.mods" : "Restituisce l'elenco delle mod in uso.",
+  "crafttweaker.command.description.patreon" : "Apre un link a Patreon",
+  "crafttweaker.command.description.recipes" : "Restituisce informazioni su tutte le ricette.",
+  "crafttweaker.command.description.recipes.hand" : "Restituisce informazioni su tutte le ricette per l'oggetto nella mano",
+  "crafttweaker.command.description.recipes.inventory" : "Restituisce informazioni su tutte le ricette per gli oggetti nell'inventario del giocatore",
+  "crafttweaker.command.description.recipes.manager" : "Restituisce informazioni sulle ricette per il gestore di ricette selezionato",
+  "crafttweaker.command.description.reload" : "Reindirizza al comando /reload",
+  "crafttweaker.command.description.script" : "Apre la cartella degli script",
+  "crafttweaker.command.description.syntax" : "Controlla la sintassi degli script",
+  "crafttweaker.command.dump.generated" : "È stato generato un elenco delle espressioni tra parentesi uncinate di tipo '%s'!",
+  "crafttweaker.command.dump.types" : "Salva tipi",
+  "crafttweaker.command.example.generated" : "Esempi salvati nella cartella 'examples' allinterno della cartella degli script. Fai clic per aprire la cartella!",
+  "crafttweaker.command.files.created" : "I file sono stati creati. Fai clic per aprire la cartella!",
+  "crafttweaker.command.hand.header.attributes" : "Attributi di",
+  "crafttweaker.command.hand.header.tags.block" : "Tag di blocchi",
+  "crafttweaker.command.hand.header.tags.item" : " Tag di oggetti",
+  "crafttweaker.command.hand.no.data" : "L'oggetto non contiene dati",
+  "crafttweaker.command.hand.no.tags" : "L'oggetto non ha tag",
+  "crafttweaker.command.help.page.info" : "Pagina %s di %s",
+  "crafttweaker.command.info.block.activated" : "La modalità di analisi dei blocchi è stata attivata. Fai clic con il tasto destro su un blocco per vedere i suoi dati.",
+  "crafttweaker.command.info.block.deactivated" : "La modalità di analisi dei blocchi è stata disattivata",
+  "crafttweaker.command.info.block.entity.data" : "Dati dell'entità-blocco: %s",
```

Comment:
Same as above: do you refer to these as "entità-blocco"?

Reply:
See above